posts/2023-09-26-23.0.0.10-beta.adoc

```diff
@@ -0,0 +1,635 @@
++  
+If the `monitorInterval` is set to `0`, the default value, the directory stops being monitored.
++
+The `ltpa.keys` file can be renamed, for example, `validation1.keys` and then the Liberty server automatically regenerates a new `ltpa.keys` file with new primary keys that are used for all new LTPA tokens created. The keys in `validation1.keys` continue to be used for validating existing LTPA tokens. 
++   
+When the `validation1.keys` are no longer needed, remove them from usage by deleting the file, or by setting `monitorDirectory` to `false`. It is recommended to remove validation keys as it can improve performance.
+
+* Specify the validation keys file and optionally specify a date-time to stop using the validation keys.
+   1. Copy the primary keys file (`ltpa.keys`) to a validation keys file, for example `validation1.keys`.
+   2. Modify the server configuration to use the validation keys file  by specifying a `<validationKeys>` server configuration element inside the `<ltpa>` element. For example, add the following configuration to the `server.xml` file:
++
+[source,xml]
+----
+<ltpa>
+    <validationKeys fileName="validation1.keys" password="{xor}Lz4sLCgwLTs=" notUseAfterDate="2024-01-02T12:30:00Z"/>
+<ltpa/>  
+----
++
+The `validation1.keys` file can be removed from usage at a specified date-time in the future with the optional attribute, `notUseAfterDate`.  It is recommended to use `notUseAfterDate` to remove validation keys after a given period as it can improve performance.
++
+The `fileName` and `password` attributes are required in the `validationKeys` element, but `notUseAfterDate` is optional.
++
+After the validation keys file is loaded from the server configuration update, the original primary keys file (`ltpa.keys`) can be deleted, which triggers new primary keys to be created while continuing to use `validation1.keys` for validation.
++
+Specifying validation keys in this way can be combined with enabling monitor directory to also use validation keys that are not specified in the `server.xml` configuration at the same time. For example:
+
+
++
+[source,xml]
+----
+<ltpa monitorDirectory="true" monitorInterval="5m">
+    <validationKeys fileName="validation1.keys" password="{xor}Lz4sLCgwLTs=" notUseAfterDate="2024-01-02T12:30:00Z"/>
+<ltpa/>
+----
++     
+To see all of the Liberty `<ltpa>` server configuration options see link:/docs/latest/reference/config/ltpa.html[LTPA configuration docs].
+    
+    
+// DO NOT MODIFY THIS LINE. </GHA-BLOG-TOPIC> 
+
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/26175
+// Contact/Reviewer: rsherget
+// // // // // // // // 
+[#dropin]
+== Allow including directories containing configDropins

```

Comment:
configDropins- is only mentioned in the heading here- what is the significance to this update? If it needs to be included we should also explain what it is since most users likely wont know.

Based on the content, i'd expect the heading to be something like "Include all files in a specified directory in your server configuration"

Reply:
Ah, I re-wrote the text based on a clearer explanation over Slack. I missed the header but I like your suggestion. Thanks.